File: library/src/main/java/cn/cqray/android/dialog/amin/ZoomIn.java
package cn.cqray.android.dialog.amin;

import androidx.annotation.NonNull;

import cn.cqray.android.anim.AnimatorBuilder;

/**
 * 缩放进入
 * @author Cqray
 */
public class ZoomIn extends DialogAnimator {

    @Override
    public void onHandle(@NonNull AnimatorBuilder builder) {
        builder.zoomIn();
    }
}

File: library/src/main/java/cn/cqray/android/dialog/amin/ZoomIn.kt
package cn.cqray.android.dialog.amin

import cn.cqray.android.anim.ViewAnimator

/**
 * 缩放进入
 * @author Cqray
 */
class ZoomIn : GetAnimator() {
    override fun onAnimatorPrepared(animator: ViewAnimator) {
        animator.zoomIn()
    }
}